tests/shutdown/shutdown.pm

```diff
@@ -17,12 +17,18 @@
 use strict;
 use base 'basetest';
 use testapi;
+use utils;
 
 sub run {
 
     select_console('root-console');
     if (get_var('DROP_PERSISTENT_NET_RULES')) {
-        type_string "rm -f /etc/udev/rules.d/70-persistent-net.rules /etc/sysconfig/network/ifcfg-eth*\n";
+        if (sle_version_at_least('12-SP2')) {
+            type_string "rm -f /etc/udev/rules.d/70-persistent-net.rules\n";
+        }
+        else {
+            type_string "rm -f /etc/udev/rules.d/70-persistent-net.rules /etc/sysconfig/network/ifcfg-eth*\n";
+        }
```

Comment:
I would recommend to put your comments in here because it will be hard to follow under which circumstances you want the if or else branch and why this depends on the SLE version.
Also, better extract the `type_string "rm -f /etc/udev/rules.d/70-persistent-net.rules\n";` and call it in both cases, e.g.

```perl
type_string "rm -f /etc/udev/rules.d/70-persistent-net.rules\n";

# <your explanation here>
unless (sle_version_at_least('12-SP2')) {
    type_string "rm -f /etc/sysconfig/network/ifcfg-eth*\n";
}
```